manifests/charts/ambient/Chart.yaml

```
@@ -0,0 +1,30 @@
+apiVersion: v2
+name: ambient
+description: Helm umbrella chart for ambient
+type: application
+version: 1.0.0
+appVersion: 1.0.0
+kubeVersion: ">= 1.23.0-0"
+keywords:
+  - istio-cni
+  - istio
+  - ambient
+sources:
+  - https://github.com/istio/istio
+icon: https://istio.io/latest/favicons/android-192x192.png
+
+# These will be stamped out to real versions during release publish
+dependencies:
+  - name: base
+    version: 1.0.0
+    repository: "file://../base"
+  - name: cni
+    version: 1.0.0
+    repository: "file://../istio-cni"
+  - name: istiod
+    version: 1.0.0
+    repository: "file://../istio-control/istio-discovery"
+    condition: enable.istiod
```

```suggestion
    condition: istiod.enabled
```
?

I think I have seen this pattern before. If I am imagining it then this works too